### PR TITLE
Remove unused GCallstacks from Capture

### PR DESCRIPTION
**Pull request by dimitry-**
callstack_hash was always 0 because of b/159107241, once
we have callstacks for uprobes we can revisit this.

### CODE REVIEW

File: OrbitClientProtos/capture_data.proto [hunk 1]
```diff
@@ -92,7 +92,7 @@ message TimerInfo {
   Type type = 6;
 
   int32 processor = 7;
-  uint64 callstack_hash = 8;
+  uint64 callstack_id = 8;
```
Comment (dimitry-) [inline OrbitClientProtos/capture_data.proto:95]: this field is now unused - I do not know if we should remove it or keep it for the future?

Comment (IrinaShkviro) [reply OrbitClientProtos/capture_data.proto:95]: I would leave this question for @pierricgimmig : he wanted to leave this field and use it in near future

Comment (IrinaShkviro) [reply OrbitClientProtos/capture_data.proto:95]: So probably just keep it for now

Comment (dpallotti) [reply OrbitClientProtos/capture_data.proto:95]: This would be used for callstacks of dynamically instrumented functions and scheduling slices I assume.

**Pull request merged**